ui/appui-react/src/appui-react/stagepanels/StagePanelDef.tsx

```diff
@@ -258,3 +290,13 @@ export const setPanelSize = produce((
   const panel = nineZone.panels[side];
   panel.size = size === undefined ? size : Math.min(Math.max(size, panel.minSize), panel.maxSize);
 });
+
+/** @internal */
+export const setPanelPinned = produce((
```

Comment:
Can you type this function and the `setPanelSize` one above ? 
```suggestion
export const setPanelPinned: (nineZone: NineZoneState, side: PanelSide, pinned: boolean) => NineZoneState = produce((
```

That should reduce the number of lines in the api .md file

Reply:
Yup, that did the trick